### ISSUE TITLE
Multiple minecarts won't show up

### ISSUE THREAD

**Issue by DR4CONS** [Andrew6rant/InventoryTabs#56]
If there is more than one minecart, only one will show up.

![2022-08-21_21 39 37](https://user-images.githubusercontent.com/90795071/185828012-def97571-6554-434d-b891-fffdfb37f46c.png)

**Comment by Andrew6rant** [comment]
Yeah, currently duplicate entities show up as one tab. That is part of the reason why 0.9 is currently a beta.

Block tabs use their coordinates to handle duplicates, but I can't do that for entities since they move. I think I have figured out a way to fix this, but I won't be able to push the code as I am going to be very busy for the next week
